src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -63,7 +69,7 @@ angular
       return @
 
     template: ->
-      @_template.get(@data.series, @options)
+      @_template.get(@data.series, @options, @data.xAxis, @data.yAxis, @data.plotOptions)
```

Comment:
Ok for now, @cesar-tonnoir will latest 1.6 be back-merged into lmi? If so there will be lots of conflicts as this whole factory has been reworked (#480) to simplify providing options.

Reply:
@xaun I don't think that will be necessary: the LMI widgets will use the figure+chart generic template (will be developed in 1.8 with your implementation of generic templates loading), so ultimately we shouldn't need that many customisations

Reply:
@cesar-tonnoir when you say "will be developed", is that different to merging this branch into upstream/1.x ? Seems there is quite a bit of HighchartsFactory#formatter customisation (of which I am not too happy with the resulting code), will the formatter stuff be discarded for generic templates? Or will it live separately somehow?